01-basics/04-MeetupTitle/script.js

```diff
@@ -14,4 +14,28 @@ function fetchMeetupById(meetupId) {
   });
 }
 
-// Требуется создать Vue приложение
+const App = {
+  data() {
+    return { meetupTitle: '', selectedId: '1' };
+  },
+
+  watch: {
+    selectedId: {
+      immediate: true,
+
+      async handler(newValue, oldValue) {
+        if (oldValue === newValue) {
+          return;
+        }
```

Comment:
Vue triggers reactivity on change only then the value is actually changed. Vue checks `oldValue === newValue` itself and in case of equality it doesn't trigger reactive effects. So this check is reduntant.